roles/common/tasks/config_validations.yml

```diff
@@ -206,3 +206,47 @@
     fail_msg: "OAuth is not supported on FIPS enabled cluster yet. Please disable FIPS or OAuth to setup the cluster."
   when: auth_mode == 'oauth' or auth_mode == 'ldap_with_oauth'
   tags: validate
+
+- assert:
+    that:
+      - sasl_protocol.replace(' ', '').split(',') | length < 3
```

Comment:
We have created a filter for spliting the protocol string to list then why not use it everywhere ?

Reply:
Done.